originals/f.yaml

```diff
@@ -303,3 +303,14 @@
     - Arcade
     subgenres:
     - Pinball
+
+- name: 'Future Cop: LAPD'
+  external:
+    wikipedia: 'Future Cop: LAPD'
+  platforms:
+  - OSX
+  - PlayStation
```

Comment:
I had not tried to compile this for the Playstation 1-5.
Also, this compiles on Linux as well.

Reply:
this is describing the platforms for the original game